Apps/W1/HybridGP/app/src/codeunits/HybridGPManagement.codeunit.al

```diff
@@ -24,6 +24,7 @@ codeunit 4016 "Hybrid GP Management"
 
         UpdateStatusOnHybridReplicationCompleted(RunId, NotificationText);
         HandleInitializationofGPSynchronization(RunId, SubscriptionId, NotificationText);
+        AttemptAutoRunDataUpgrade();
```

Comment:
We are not giving option to not auto run dataupgrade?

Reply:
The setting hasn't been created yet, we haven't decided where this will go. It will be there, just not in this PR. This feature likely won't be submitted to MSFT until that is put in place, with input from Nikola. I believe he has started a migration-wide settings table, but it hasn't been pulled into the public repo. Once this is done, the setting will probably go there.